lib/tasks/ga4gh_wes.rake

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+namespace :ga4gh_wes do
+  # the following can defined in credentials to configure the client
+  # ga4gh_wes:
+  #   oauth_token: <some oauth token>
+  #   headers: { '<some header key>': '<some header value>' }
+  #   server_url_endpoint: 'https://localhost:7500/ga4gh/wes/v1/'
```

Comment:
Not a big deal, but I copied this line to test against docker container. Should be `http://localhost:7500/ga4gh/wes/v1/`.